constants.py

```diff
@@ -0,0 +1,53 @@
+import os
```

Comment:
This should be moved to the `examples` directory.

Reply:
done.